setup.py

```diff
@@ -55,9 +55,21 @@ def get_version():
         for line in lines:
             if line.startswith('Version:'):
                 return line[8:].strip()
-    if os.path.exists("./tools"):
-        print("Running tools/get_version.sh")
-        version = subprocess.getoutput("bash -c 'cd tools; ./get_version.sh'")
+    if os.path.exists("./include/vrv"):
+        with open("./include/vrv/vrvdef.h") as header_file:
+            defines = {}
+            for line in header_file.readlines():
+                if line.startswith("#define"):
+                    line.rstrip()
+                    definition = line[8:].split()
```

Comment:
There's not really a whole lot of difference between splitting from `#defines` and not -- you can just do stuff with a 3-size list, rather than a 2-size list.